lib/govuk_publishing_components/presenters/taxonomy_list_helper.rb

```diff
@@ -14,7 +14,7 @@ def image_card_data
             image_src: image_card[:image][:url],
             image_alt: image_card[:image][:alt],
             heading_level: image_card[:link][:heading_level],
-            href_data_attributes: (image_card[:link][:data_attributes] if image_card[:link][:data_attributes])
+            href_data_attributes: (image_card[:link][:data_attributes])
```

Comment:
Not a big issue, but looks like the `(` `)` brackets are redundant here.

Reply:
Thanks - had missed that 👍

Reply:
Fixed in 40eeedc